### ISSUE TITLE
Non-conv architectures do not get reported performance in paper

### ISSUE THREAD

**Issue by luke-mcdermott-mi** [yolky/RCIG#3]
I am using the script in the readme, but changed the eval_arch to resnet and vgg. These do not get reported accuracy from the paper. 

``` python3 eval.py --dataset_name cifar10 --checkpoint_path ./distilled_images_final/0/cifar10_10/checkpoint_10000 --config_path ./configs_final/depth_3.txt --random_seed 0 --eval_arch resnet```

Resnet achieved 20% and VGG gets 40%. After changing the --normalization to batch, it still gets low performance. 

In the supplemental section, you state that no data augmentation gets better results for cifar10; however, your configs for cifar10 use data augmentation. Why is this? Furthermore, what hyperparameters and I supposed to use for ResNet on cifar10? I am assuming its still depth_3.txt

**Comment by yolky** [comment]
For the experiments  in the paper we used the command:

 ```python3 eval.py --dataset_name cifar10 --checkpoint_path ./distilled_images_final/0/cifar10_10/checkpoint_10000 --config_path ./configs_final/depth_3.txt --random_seed 0 --eval_arch resnet --eval_lr .0003```

This should achieve around 51.3% **with data augmentation during evaluation** (note that our eval script first evaluates without data augmentation then with data augmentation). We found that for cross architecture generalization with ResNets using data agumentation significantly improves performance. This corresponds to row 6 in table 3 in the paper (RCIG - Conv-NN).

Similarly, 

``` python3 eval.py --dataset_name cifar10 --checkpoint_path ./distilled_images_final/0/cifar10_10/checkpoint_10000 --config_path ./configs_final/depth_3.txt --random_seed 0 --eval_arch resnet --eval_lr .0003 --normalization batch```

should achieve 49.7%.

Our eval script works by first evaluating it without data augmentation, then with data augmentation (using whatever augmentation is in the config). So our evaluation scripts outputs two accuracies: one without DA and one with DA.

For the 54.4% and 54.8% numbers in the paper, we used batchnorm during distillation (RCIG - Conv-BN). The pre-distilled dataset in the repo was distilled without batchnorm.

Yes you should use depth_3.txt for the resnet.

**Comment by luke-mcdermott-mi** [comment]
Thanks!
